lib/util.js

```diff
@@ -21,6 +21,7 @@ util.parseCommandLineArgs = defaults => {
       `Directory with the tapes in HAR format. defaults to ${defaults.tapeDir}`,
       util.formatPath, defaults.tapeDir)
     .option('-r, --record', 'Erase all files in tapeDir and record new tapes')
+    .option('-v, --verbose', 'Outputs verbose log information')
```

Comment:
we should also include the default here:
`.option('-v, --verbose', 'Outputs verbose log information', defaults.verbose)`